Extensions/Content/Dnn.PersonaBar.Pages/Components/IPagesController.cs

```diff
@@ -23,7 +23,7 @@ public interface IPagesController
 
         TabInfo MovePage(PageMoveRequest request);
 
-        void DeletePage(PageItem page, PortalSettings portalSettings = null);
+        void DeletePage(PageItem page, PortalSettings portalSettings = null, bool hardDelete = false);
```

Comment:
@zyhfish please make the new method an overload

Reply:
updated.